examples/tutorials/analysis-1d/spectral_analysis_rad_max.py

```diff
@@ -318,7 +318,7 @@
 
 ######################################################################
 # For more ways of assessing fit quality, please refer to the dedicated
-# `modeling and fitting tutorial :doc:`/tutorials/api/fitting` tutorial.
+# :ref:`modeling and fitting </tutorials/api/fitting>` tutorial.
```

Comment:
this would read in the documentation as `, please refer to the dedicated modeling and fitting Fitting tutorial.` so maybe the last line can be left as:
```
:doc:`/tutorials/api/fitting` tutorial.
```